testsuite/src/main/java/io/netty/testsuite/transport/AbstractSingleThreadEventLoopTest.java

```diff
@@ -0,0 +1,38 @@
+package io.netty.testsuite.transport;
+
+import org.junit.Test;
+
+import io.netty.channel.Channel;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.SingleThreadEventLoop;
+import io.netty.channel.socket.ServerSocketChannel;
+
+import static org.junit.Assert.*;
+
+public abstract class AbstractSingleThreadEventLoopTest {
+
+  @Test
+  public void testChannelsRegistered()  {
+    EventLoopGroup group = newEventLoopGroup();
+    final SingleThreadEventLoop loop = (SingleThreadEventLoop) group.next();
+
+    try {
+      final Channel ch1 = newChannel();
```

Comment:
@vkostyukov please use 4 spaces everywhere to match our coding style.

Reply:
Oh sorry about this. I'm used to formatters doing this automagically. The change is coming.

Reply:
Done.